docs/cookbook/extending-product-list.md

```diff
@@ -66,9 +66,123 @@ class ListedProductView extends BaseListedProductView
 }
 ```
 
-### 2. Extend `ListedProductViewFactory` so it returns the new required data
+### 2. Add new attribute to ElasticSearch
+
+In order to add new attribute to ElasticSearch you need to add it to the structure first.
+You can do that by adding it to `mappings` in all `src/Shopsys/ShopBundle/Resources/definition/product/*.json` files like this:
+```json
+// ...
```

Comment:
well technically in JSON are no comments and it's really weirdly formatted on GitHub.
What about 
```json
  "mappings": {
    "_doc": {
      "properties": {
        "brand_name": {
          "type": "text"
        },
```

Do you think it's understandable?